### DISCUSSION TITLE
Social cards does not work:

### DISCUSSION THREAD

**Discussion by jonaprieto** [squidfunk/mkdocs-material#5396] [Q&A]
I tried to follow the instructions
here: https://squidfunk.github.io/mkdocs-material/setup/setting-up-social-cards/?h=social
but it seems that the PNG files are not being generated for the social cards.
How can I fix/check/debug the social cards? 

![image](https://user-images.githubusercontent.com/1428088/233342208-5fdbb109-d42b-4c7c-ab06-02102384875b.png)

**Comment by jonaprieto** [comment]
For some pages, when I copy the URL and past it, e.g. on Discord, they work, but not for all the pages, or the main page.

**Reply by squidfunk** [reply]
We're happy to help you if you can provide a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/). Without such a reproduction we can only guess.

**Comment by kamilkrzyskow** [comment]
Hi @jonaprieto 

To check what images are generated you can check this path https://github.com/anoma/juvix-docs/tree/gh-pages/0.3.2/assets/images/social. I assume it's the docs you're asking about. 
Then you can check the `<head>` section of a given URL to see what image URL was assigned to it.

For the main page https://docs.juvix.org/0.3.2/
it's the link https://docs.juvix.org/latest/assets/images/social/index.png
it's 404, so perhaps the Social Plugin doesn't handle mike versioning? 🤷 

As for the missing image on Discord, I had it too when testing some time ago, it fixed itself after a few hours (for actually working images 😏), maybe Discord caches results and doesn't load the head of the page after sending a link, which is weird imo, or maybe GitHub pages cache something else. ✌️
